packages/now-cli/src/commands/deploy/legacy.ts

```diff
@@ -1228,7 +1062,7 @@ function getVerifyDCsGenerator(
   const verifyDeployment = verifyDeploymentScale(
     output,
     now,
-    deployment.deploymentId,
+    deployment.deploymentId || deployment.uid,
```

Comment:
Why is this line changing? In what scenario would the deploymentId be missing?

Reply:
Some v1 deployments didn't have it and had `uid` instead